### PR TITLE
[release-1.11] image_pull: fix variable shadowing

### PR DESCRIPTION
**Pull request by runcom**
Signed-off-by: Antonio Murdaca <runcom@redhat.com>

**- What I did**

fixes the various failures we've seen in the CI in #1772 and #1768 

**- How I did it**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

**Comment by runcom** [comment]
@mrunalp PTAL

master commit is embedded into #1772

File: pkg/storage/image.go [hunk 1]
```diff
@@ -86,7 +86,7 @@ type ImageServer interface {
 	ImageStatus(systemContext *types.SystemContext, filter string) (*ImageResult, error)
 	// PrepareImage returns an Image where the config digest can be grabbed
 	// for further analysis. Call Close() on the resulting image.
-	PrepareImage(systemContext *types.SystemContext, imageName string, options *copy.Options) (types.Image, error)
+	PrepareImage(imageName string, options *copy.Options) (types.Image, error)
```
Comment (mrunalp) [inline pkg/storage/image.go]: All the other methods are taking a context vs this one not taking anymore.

Comment (runcom) [reply pkg/storage/image.go]: yup, it's not needed in this one as it was not in the old `CanPull`

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**